supervision/keypoint/annotate.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional, Tuple
+
+import cv2
+import numpy as np
+
+from supervision.annotators.base import ImageType
+from supervision.draw.color import Color
+from supervision.keypoint.core import KeyPoints
+from supervision.keypoint.skeletons import resolve_skeleton_by_vertex_count
+from supervision.utils.conversion import convert_for_annotation_method
+
+
+class BaseKeyPointAnnotator(ABC):
+    @abstractmethod
+    def annotate(self, scene: ImageType, detections: KeyPoints) -> ImageType:
+        pass
+
+
+class VertexAnnotator(BaseKeyPointAnnotator):
+    def __init__(
+        self,
+        color: Color = Color.ROBOFLOW,
+        radius: int = 4,
+    ) -> None:
+        """
+        Most basic keypoint annotator.
+
+        Args:
+            color (Color, optional): The color of the keypoint.
+            radius (int, optional): The radius of the keypoint.
+        """
+        self.color = color
+        self.radius = radius
+
+    @convert_for_annotation_method
+    def annotate(self, scene: ImageType, keypoints: KeyPoints) -> ImageType:
+        if len(keypoints) == 0:
+            return scene
+
+        xy_all = keypoints.xy
+        for xy in xy_all:
```

Comment:
I think we could just do `for xy in keypoints.xy` here.

Reply:
Yup